### ISSUE TITLE
List of available algorithms

### ISSUE THREAD

**Issue by rubenSaro** [bd2kccd/r-causal#98]
Hi @chirayukong,

Is there a function to display the list of available algorithms for r-causal ?
Also, does the current version of r-causal, correspond to the most recent Tetrad version ?

Thanks,

Ruben

**Comment by chirayukong** [comment]
Yes, there is a function showing the list of available in r-causal but I can't remember at the moment. The current state of r-causal is not up-to-date. I will update the jar files in the weekend. I'll let you know when it's done and what the function is.

**Comment by OnkarMulay** [comment]
Can you let us know now? Have you updated?
